src/answers-headless.ts

```diff
@@ -261,7 +271,12 @@ export default class AnswersHeadless {
    *
    * @returns A Promise of a {@link UniversalSearchResponse} from the Answers API
    */
-  async executeUniversalQuery(): Promise<UniversalSearchResponse> {
+  async executeUniversalQuery(): Promise<UniversalSearchResponse | undefined> {
+    if(this.state.meta.searchType !== SearchTypeEnum.Universal) {
```

Comment:
Does `executeUniversalAutocomplete` also need to have this console error added?

Reply:
Since UniversalAutocomplete has no impact on the state, there's not really a reason to prevent it from being ran on a vertical. The autocomplete functions are more stateless than regular universal and vertical search. Autocomplete for vertical and filter search require a verticalKey so an error there is required